gstreamer/src/format.rs

```diff
@@ -12,6 +12,7 @@ use ClockTime;
 use Format;
 
 #[derive(PartialEq, Eq, Hash, Clone, Copy, Debug)]
+#[cfg_attr(feature = "ser_de", derive(Serialize, Deserialize))]
```

Comment:
Ah interesting, I would've just handled this like a `(gst::Format, i64)` :)

Reply:
Let me take another look at this.

Reply:
No it's fine as-is, I just didn't think about the simple solution :)

Reply:
Unless you think it leads to nicer serialization?

Reply:
Yes, I think we can do better than the `((things))` anyway.